config/samples/cassandra.datastax.com_v1beta1_cassandradatacenter.yaml

```diff
@@ -1,7 +1,29 @@
 apiVersion: cassandra.datastax.com/v1beta1
 kind: CassandraDatacenter
 metadata:
-  name: cassandradatacenter-sample
+  name: dc2
```

Comment:
Should we choose a different name for this sample?

Reply:
Yes, do you have a proposal? dc1 & cluster1? Or something fancier?

Reply:
I set them to dc1 & development for now..

Reply:
Great!